src/solidity-frontend/solidity_grammar.cpp

```diff
@@ -252,6 +260,14 @@ const char *elementary_type_name_to_str(ElementaryTypeNameT type)
   }
 }
 
+unsigned int uint_type_name_to_size(ElementaryTypeNameT type)
+{
+  if(!uint_size_map.count(type))
+    assert(!"Size map missing for type");
```

Comment:
when building with -DNDEBUG flag, this assertion will be removed, and the if guard will be applied to the return statement. Please use msg.error + abort here.

Reply:
The `.at()` will throw in that case, so I don't think the msg.error + abort() is needed here.